src/upload.ts

```diff
@@ -257,23 +276,26 @@ export async function uploadLargeFileRequest(
   let splitSizeIntoParts:
     | ((totalSize: number, partCount: number) => Array<{ start: number; end: number }>)
     | undefined = undefined;
+  const chunkSize = TUS_CHUNK_SIZE * opts.chunkSizeMultiplier;
```

Comment:
nit: Should we check whether the multiplier is an integer? We don't have type guarantees in JS. But this might be a little too paranoid of me. Feel free to resolve without action.

Reply:
That's a good point, it's really crucial that we only use strict multiples of the base chunk size.